@utahdts/utah-design-system/react/contexts/UtahDesignSystemContext/hooks/useBanner.js

```diff
@@ -0,0 +1,66 @@
+// @ts-check
+import { useCallback, useMemo } from 'react';
+import useUtahDesignSystemContext from '../useUtahDesignSystemContext';
+import uuidv4 from '../../../util/uuidv4';
+
+/** @typedef {import('../../../jsDocTypes').UtahDesignSystemContextBanner} UtahDesignSystemContextBanner */
+/** @typedef {import('../../../jsDocTypes').BannerPlacement} BannerPlacement */
+/**
+ * @returns {{
+ * addBanner: function({
```

Comment:
I don't think multiline jsdoc works... (typescript does work that way... :|) 

@returns {{addBanner: (banner: UtahDesignSystemContextBanner) => void, removeBanner: (banner: UtahDesignSystemContextBanner) => void}}

Reply:
KISS and leave it
or
Modify type so that "id" is optional.
or
Create a new type that doesn't include id.
or
Create a partial type
/** @type {Partial<SettingsInput>} */
This has the drawback that makes required as optional 😞